app/models/grda_warehouse/warehouse_clients_processed.rb

```diff
@@ -304,6 +304,7 @@ def as_hash
         active_in_cas_match: client.cas_reports.where(active_match: true).exists?,
         last_exit_destination: client.last_exit_destination,
         last_cas_match_date: client.cas_reports.maximum(:match_started_at),
+        lgbtq_from_hmis: client.hmis_client&.sexual_orientation
```

Comment:
This isn't going to work, unless I'm missing something.  client is a destination client. So this would need to be something like client&.source_hmis_clients&.order(updated_at: :desc)&.first&.sexual_orientation...

Reply:
I'm moving the logic over to the client model, seems cleaner.